lib/cli.js

```diff
@@ -0,0 +1,101 @@
+import {Command, Option} from 'commander'
+import {SyntaxError, VALID_OUTPUTS, draw} from './index.js'
+import {name, version} from '../package.js'
+import fs from 'fs'
+import l4js from 'log4js'
+import path from 'path'
+
+const log = l4js.getLogger()
```

Comment:
I'm now questioning the utility of all of this logging.  Perhaps it should all be ripped out?  Add an issue to discuss.

Reply:
Added #31